src/app/developer/components/Books.tsx

```diff
@@ -24,11 +24,22 @@ class Books extends React.Component<Props, State> {
     const {archiveLoader, osWebLoader} = this.props.services;
     const bookLoader = makeUnifiedBookLoader(archiveLoader, osWebLoader);
 
-    const books = await Promise.all(Object.entries(BOOKS).map(([bookId, {defaultVersion}]) =>
-      bookLoader(bookId, defaultVersion)
-    ));
-
-    this.setState({books});
+    const sortBooks = (bookA: Book, bookB: Book) => {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare does this in one line